src/routes/conversation/[id]/+page.svelte

```diff
@@ -75,20 +75,10 @@
 			loading = true;
 			pending = true;
 
-			const module = await import("browser-image-resizer");
-			// currently, only IDEFICS is supported by TGI
-			// the size of images is hardcoded to 224x224 in TGI
-			// this will need to be configurable when support for more models is added
-			const resizedImages = await Promise.all(
-				files.map(async (file) => {
-					return await module
-						.readAndCompressImage(file, {
-							maxHeight: 224,
-							maxWidth: 224,
-							quality: 1,
-						})
-						.then(async (el) => await file2base64(el as File));
-				})
+			const base64Files = await Promise.all(
+				(files ?? []).map((file) =>
+					file2base64(file).then((value) => ({ type: "base64" as const, value, mime: file.type }))
+				)
 			);
 
```

Comment:
I created a more generic image processor in https://github.com/huggingface/chat-ui/pull/1021/commits/c8814f4b8c545ec4b4f356ddba3e58dd34c0d7e2 and put the multimodal options at `endspoints[*].multimodal`. What do you think?

Reply:
WRT to bandwidth, it might make sense to do some image processing on the server on upload to convert to a more suitable format (i.e. put everything in AVIF/WEBP?) but I found the lower upload limit to get in the way